content/css/concepts/animations/animations.md

```diff
@@ -38,5 +38,6 @@ h1 {
   color: yellow;
   animation-name: color-change;
   animation-duration: 5s;
+  transition: 5s;
 }
```

Comment:
```suggestion
/* The animation */
@keyframes color-change {
  from {
    color: purple;
  }
  25% {
    color: yellow;
  }
  50% {
    color: red;
  }
  75% {
    color: blue;
  }
  to {
    color: yellow:
  }
}

/* The element */
h1 {
  color: purple;
  animation-name: color-change;
  animation-duration: 5s;
}
```

We should edit the `@keyframes` example to highlight how CSS animations allow multiple style changes between frames.